db/re/item_db_usable.yml

```diff
@@ -12610,7 +12610,7 @@ Body:
       NoMail: true
       NoAuction: true
     Script: |
-      itemskill "AC_CONCENTRATION",max(getskilllv("AC_CONCENTRATION"),3);
+      bonus_script "{ bonus bBaseAtk,30; bonus bMatk,30; bonus bAspdRate,5; bonus bVariableCastrate,-5; bonus bMaxHPrate,-10;  bonus bMaxSPrate,-10; bonus5 bAutoSpell,\"AC_CONCENTRATION\",max(getskilllv(\"AC_CONCENTRATION\"),3),10,BF_WEAPON|BF_MAGIC,0; }",600,1,1,EFST_2011RWC_SCROLL;
```

Comment:
Please implement new statuses in status.yml, bonus_script is less reliable

Reply:
Thanks!